### PR TITLE
Added grafana

### PR DESCRIPTION
**Pull request by cornet**

### CODE REVIEW

File: grafana/Makefile [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+LICENSE=	apache-2.0
+
+BUILD_DEPENDS+=		go-[0-9]*:../../lang/go
+USE_LANGUAGES=		c # go 
+NO_CONFIGURE=		yes
+
+INSTALLATION_DIRS=	bin
+
+MAKE_ENV+=		GOPATH=${WRKDIR}/gopath
+INSTALL_UNSTRIPPED=	yes
```
Comment (stevenwilliamson) [inline grafana/Makefile]: Why is this needed, it should be the default iirc ?

Comment (stevenwilliamson) [inline grafana/Makefile]: Actually if it isn't the default for our platform we should make it so by setting it in make.conf stripping is bad

File: grafana/Makefile [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+
+SUBST_CLASSES+=		version
```
Comment (stevenwilliamson) [inline grafana/Makefile]: Comment would be useful to explain what we are fixing with the sub in-case it gets fixed in the future, make it obvious what it's for why it was needed

File: grafana/PLIST [hunk 3]
```diff
@@ -0,0 +1,2 @@
+@comment $NetBSD$
+bin/grafana-server
```
Comment (stevenwilliamson) [inline grafana/PLIST]: yay for go binaries

File: grafana/Makefile (hunk 1 above)
Comment (stevenwilliamson) [inline grafana/Makefile]: just to note why stripping is bad, it makes debugging a complete PITA by stripping all the type information and function names

Comment (cornet) [inline grafana/Makefile]: The default is NULL and it is def stripping by default.

Agreed we probably should make it default but still should be added here as you shouldn't strip go binaries : https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=717172

File: grafana/Makefile [hunk 4]
```diff
@@ -0,0 +1,58 @@
...
+
+MAKE_ENV+=		GOPATH=${WRKDIR}/gopath
+
+GRAFANA_USER?=  grafana
+GRAFANA_GROUP?= grafana
+
+PKG_USERS_VARS+=  GRAFANA_USER
+PKG_GROUPS_VARS+= GRAFANA_GROUP
+PKG_GROUPS=       ${GRAFANA_GROUP}
+PKG_USERS=        ${GRAFANA_USER}:${TINYPROXY_GROUP}
```
Comment (stevenwilliamson) [inline grafana/Makefile]: TINYPROXY ?

**Comment by stevenwilliamson** [comment]
:+1:
